model/project_parser.go

```diff
@@ -643,6 +627,7 @@ const (
 type GetProjectOpts struct {
 	Ref          *ProjectRef
 	PatchOpts    *PatchOpts
+	LocalModules *map[string]string
```

Comment:
We don't typically make maps pointers (not really a reason to)

Reply:
i thought it would be easier and faster to check this instead of an empty map

Reply:
This doesn't look updated